samples/scala-reliable-timers/src/main/scala/com/example/actions/OrderAction.scala

```diff
@@ -0,0 +1,122 @@
+package com.example.actions
+
+import java.util.UUID
+
+import com.example.domain.Order
+import io.grpc.Status
+import io.grpc.StatusRuntimeException
+import org.slf4j.LoggerFactory
+
+// FIXME: we need to provide a default dispatcher for users to use
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.concurrent.duration.DurationInt
+
+import akka.Done
+import com.example
+import com.example.OrderStatus
+import com.google.protobuf.empty.Empty
+import kalix.scalasdk.action.Action
+import kalix.scalasdk.action.ActionCreationContext
+// This class was initially generated based on the .proto definition by Kalix tooling.
+//
+// As long as this file exists it will not be overwritten: you can maintain it yourself,
+// or delete it so it is regenerated as needed.
+
+// tag::timers[]
+class OrderAction(creationContext: ActionCreationContext) extends AbstractOrderAction {
+// end::timers[]
+  val logger = LoggerFactory.getLogger(classOf[Order])
+
+  // tag::place-order[]
+
+  def timerName(orderNum: OrderNumber) =
+    "order-expiration-timer-" + orderNum.number
+
+  override def placeOrder(orderRequest: OrderRequest): Action.Effect[OrderNumber] = {
+
+    val orderNumber = OrderNumber(UUID.randomUUID().toString) // <1>
+
+    val timerRegistration: Future[Done] = // <2>
+      timers.startSingleTimer(
+        name = timerName(orderNumber), // <3>
+        delay = 5.minutes, // <4>
```

Comment:
This is made 5 minutes to make it more realistic for the docs, but not so convenient if someone wants to play around with the sample 🤷